src/screens/safe-overview/components/danger-zone/danger-zone.component.styles.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Button } from 'components/primitive';
+
+export const StyledRow = styled.span`
```

Comment:
Why are we not using `Box` here?

Reply:
my bad- changes were in the staging area. pushed the latest code.